### ISSUE TITLE
Null result from `aiImportFileFromMemory()`

### ISSUE THREAD

**Issue by controllerface** [LWJGL/lwjgl3#990]
### Version

3.3.4 (snapshot)

### Platform

Windows x64

### JDK

OpenJDK 22

### Module

Assimp

### Bug description

Tried out the latest snapshot, and found that I was getting an NPE in my model loader, so I whipped up a very basic test with some `assert`s to verify it in isolation, and was able to replicate the issue. I am calling `aiImportFileFromMemory()` to load a model file, which works in `3.3.3` but returns null in the latest snapshot. 

I have attached the test case in a zip file, it's just a simple Main class and a test model, as well as the gradle script. Just to double check it works correctly in the current stable release, I tried changing the lwjgl version to `3.3.3` and doing this with no other changes, the test passes with no issues. 

Pasted stack trace is pasted from my actual project, not the simple test, but is pretty short so hopefully helps track down the issue. I did try setting breakpoints inside `Assimp.class`  and verified that on this line: 
```java
long __functionAddress = Assimp.Functions.ImportFileFromMemory;
```  
the `__functionAddress` variable is non-zero, so it does look like the library is loaded, but for some reason the internal call to `naiImportFileFromMemory()` returns a value of `0`. That was as far as I went on my own.

[assimp_bug.zip](https://github.com/user-attachments/files/15893768/assimp_bug.zip)

### Stacktrace or crash log output

```shell
Exception in thread "main" java.lang.RuntimeException: Unable to load model data
	at com.controllerface.bvge.geometry.ModelRegistry.load_model(ModelRegistry.java:145)
	at com.controllerface.bvge.geometry.ModelRegistry.load_model(ModelRegistry.java:65)
	at com.controllerface.bvge.geometry.ModelRegistry.init(ModelRegistry.java:765)
	at com.controllerface.bvge.game.TestGame.<init>(TestGame.java:82)
	at com.controllerface.bvge.window.Window.init_game_mode(Window.java:331)
	at com.controllerface.bvge.Main.main(Main.java:38)
Caused by: java.lang.NullPointerException: Cannot invoke "org.lwjgl.assimp.AIScene.mNumMeshes()" because "ai_scene" is null
	at com.controllerface.bvge.geometry.ModelRegistry.load_model(ModelRegistry.java:106)
	... 5 more
```

**Comment by controllerface** [comment]
Just a note, I also tried earlier 3.3.4 snapshots, with the same results. here's the artifacts I tried, just for clarity:
```
3.3.4-20231110.001200-1
3.3.4-20231122.134555-2
3.3.4-20231122.151814-3
3.3.4-20231218.151521-4
```

**Comment by Spasi** [comment]
Hey @controllerface,

Looks like https://github.com/assimp/assimp/issues/5585. I patched `FBXImporter.cpp` for now and importing your test scene works. The fix will be available in the next 3.3.4 snapshot. Alternatively, you could download the latest Assimp build from https://www.lwjgl.org/browse/nightly and use it with `-Dorg.lwjgl.assimp.libname=<path>` or `Configuration.ASSIMP_LIBRARY_NAME.set("<path>")`.

**Comment by controllerface** [comment]
Awesome, thanks for the quick attention. I will wait for the next snapshot and give it another go in my project.
